crypto/ledger_test.go

```diff
@@ -79,7 +79,8 @@ func TestPublicKeyHDPath(t *testing.T) {
 		// Store and restore
 		serializedPk := priv.Bytes()
 		require.NotNil(t, serializedPk)
-		require.Equal(t, 44, len(serializedPk))
+		fmt.Print(len(serializedPk))
```

Comment:
do we need this?

Reply:
nah, not really! it is a debugging left over